README.md

```diff
@@ -72,6 +72,25 @@ Enums.Framerate framerate = timecodeObj.Framerate;
 string timecodeRegex = Timecode.RegexPattern;
 ```
 
+### Operator Overloading
+
+```csharp
+
+var tc1 = new Timecode(1, 2, 20, 100, DotnetTimecode.Enums.Framerate.fps23_976);
+var tc2 = new Timecode(0, 22, 31, 100, DotnetTimecode.Enums.Framerate.fps23_976);
+
+
```

Comment:
For consistency, let's remove this extra line.

Reply:
Done. Please check and let me know if any changes are required.